Packages/com.unity.inputsystem/Documentation~/Videos.md

```diff
@@ -0,0 +1,71 @@
+---
+uid: input-system-videos
+---
+# Videos
+
+This page provides a collection of video resources to help you quickly get started with the Input System. The Input System supports several different workflows, each with its own advantages. These videos walk you through the most commonly used workflow and offer practical guidance to help you begin using the Input System right away.
+
+
+## Input System in Unity 6 (1/7): Input Action Editor
+
+**Description**: This is the first video in a 7-part series that will show you how to get started with Unity’s Input System and introduce some of its core concepts. You’ll learn how to install the Input System into your project, understand the concept of Input Actions, and how to use the Input System Actions Editor to bind them to various inputs, such as keys, gamepad buttons, and mouse axes.
+
+We’ll also explore how Interactions define when an input action is triggered (e.g., press, hold, or release) and how Processors can modify or transform raw input data before it’s used (e.g., normalizing, inverting, or clamping values).
+
+**Link**: [Watch Video](https://www.youtube.com/watch?v=TiTKAseu17A)
+
+
+## Input System in Unity 6 (2/7): Input System Scripting
+
+**Description**: In this video, where we dive into scripting with the Input System to control a third-person character.
+
+You'll learn how to write code to move and jump the third-person character using an Input System Asset, with support for both gamepad and mouse-and-keyboard inputs.
+
+We’ll also add a simple pause menu to demonstrate how Action Maps can be switched dynamically to handle different input contexts.
+
+**Link**: [Watch Video](https://www.youtube.com/watch?v=Cd2Erk_bsRY)
+
+
+## Input System in Unity 6 (3/7): Input System Mobile controls
+
+**Description**: In this video, we’ll show you how to add mobile touch controls to your game using Unity’s Input System.
+
+You’ll learn how to use the On-Screen Stick and On-Screen Button components to set up on-screen controls for mobile devices, enabling touch-based input for third-person character movement.
+
+**Link**: [Watch Video](https://www.youtube.com/watch?v=aI-r7ILNDug)
+
+
+## Input System in Unity 6 (4/7): Input System and UI toolkit
```

Comment:
In this case, I would keep Input System but switch them. I think it's easier to scan through the headings and find the content you are looking for that way:

> `## UI toolkit with the Input System`